myapp/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,65 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+  end
+
+  def new
+    @task = Task.new()
```

Comment:
引数をいれない場合のメソッドの後ろに、`()`は基本的に不要です。

```suggestion
    @task = Task.new
```

Reply:
()を削除しました。
`def new
    @task = Task.new
  end`